homeassistant/components/bsblan/models.py

```diff
@@ -0,0 +1,12 @@
+"""Models for the BSB-Lan integration."""
+
+from dataclasses import dataclass
+
+from bsblan.models import State
+
+
+@dataclass
+class BSBLanCoordinatorData:
```

Comment:
Why do we store this here instead of in the coordinator?

Reply:
Why not? It looks clean to store all data classes in models

Reply:
Because it's the data of the coordinator so it looks more logical to have it in the coordinator